cockatrice/src/window_main.cpp

```diff
@@ -795,3 +799,8 @@ void MainWindow::refreshShortcuts()
     aExit->setShortcuts(settingsCache->shortcuts().getShortcut("MainWindow/aExit"));
     aCheckCardUpdates->setShortcuts(settingsCache->shortcuts().getShortcut("MainWindow/aCheckCardUpdates"));
 }
+
+void MainWindow::notifyUserAboutUpdate()
+{
+    QMessageBox::information(this, tr("Information"), tr("A new version of the cockatrice client is available for download."));
```

Comment:
Im not sure I like the idea of presenting a URL, whats others thoughts?  Obviously we can do it.  Seems to me that the user of the client connecting should be smart enough to figure out were to go get there software updates from.